cancermuts_streamlit/readme.txt

```diff
@@ -0,0 +1,25 @@
+

```

Comment:
this should be called `README.md` (.md files are formatted according to [GitHub markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) on the website)

this would be better divided in subsections. e.g.:

- Introduction (a small description of what the web app is)
- requirements (all of them including versions they were tested on)
- usage (how to use the app)

also GitHub markdown allows to separate regular test from e.g. commands and variable names/contents, and in general to format the document in sections etc

Reply:
Fixed